README.md

```diff
@@ -13,25 +44,77 @@ git clone https://github.com/klavenj/open-forum-backend.git
 Go into the project directory
 
 ```bash
+
 cd open-forum-backend/
 ```
 
 Checkout working branch
 
 ```bash
+
 git checkout -b <branch>
 ```
 
 Install NPM dependencies
 
 ```bash
+
 npm i
 ```
 ---
 
-## Environment Variables
 
-For local developmebt, You can duplicate .env.example located file and rename it to .development.env or .env
+
+# Environment Set Up
+
+
+## Set up Database
+
+### Start Postgres (If not using the Postgres.app)
+
+```bash
+
+brew services start postgres
+```
+
+### Create Database
+
+
+In order to run the project locally with ``TypeORM`` requires an existing database. To create a database with Postgres. You can run this command to create a database.
+
+
+Run this command when you arent connected to postgres via Terminal:
+
+```bash
+
+createdb -U <username> <databasename>
```

Comment:
It seems like even I create a DB here it's not configured to be run by the project.